web/src/search_pill.ts

```diff
@@ -121,6 +122,14 @@ function search_user_pill_data_from_term(term: NarrowTerm): SearchUserPill {
     return search_user_pill_data(users, term.operator, term.negated ?? false);
 }
 
+function is_sent_by_me_pill(pill: SearchUserPill): boolean {
+    return (
+        pill.operator === "sender" &&
+        pill.users.length === 1 &&
+        pill.users[0]!.email === state_data.current_user.email
```

Comment:
`the(pill.users)` might be better than the type assertion

Reply:
yes! thanks